views/views.xml

```diff
@@ -1,48 +0,0 @@
-<odoo>
-  <data>
-    <!-- explicit list view definition -->
-
-
-<!-- actions opening views on models -->
-<!--
-    <record model="ir.actions.act_window" id="sb_purchase.action_window">
-      <field name="name">sb_purchase window</field>
-      <field name="res_model">sb_purchase.sb_purchase</field>
-      <field name="view_mode">tree,form</field>
-    </record>
--->
-
-<!-- server action to the one above -->
-<!--
-    <record model="ir.actions.server" id="sb_purchase.action_server">
-      <field name="name">sb_purchase server</field>
-      <field name="model_id" ref="model_sb_purchase_sb_purchase"/>
```

Comment:
Let me know if you wanted to keep those stuff.. (They can always be found later by checking the commits/this PR if needed).

I cleaned multiple stuff in this commit, including removing unused /commented code. You decide

Reply:
Mhmm, don't recall what it was about.
It can be cleaned. Thanks